### PR TITLE
content/en: Minor updates to the deprecated API troubleshooting documentation

### PR DESCRIPTION
**Pull request by timflannagan**
Follow-up to #173 

- Update `offical` -> `official`.
- Update tense.
- Fix various markdown violations (file contains newline, newline
  between header and content).

### CODE REVIEW

File: content/en/docs/Tasks/Troubleshooting/subscription.md [hunk 1]
```diff
@@ -30,15 +30,16 @@ If the `Status` block does not provide enough information, check the [Catalog op
...
+This error indicates that the API apiextensions.k8s.io/v1beta1, Kind=CustomResourceDefinition is not available on-cluster. It indicates this particular GVK is not present on the api-server, which will happen on Kubernetes 1.22+ with deprecated built-in types like v1beta1 CRDs or v1beta1 RBAC.
 
-This error can also arise when installing operator bundles with CustomResources that OLM supports, such as `VerticalPodAutoscalers` and `PrometheusRules`, but the relevant CustomResourceDefinition has not yet been installed. In this case, this error should eventually resolve itself provided the required CustomResourceDefinition gets installed on the cluster and is accepted by the api-server. 
\ No newline at end of file
+This error can also arise when installing operator bundles with CustomResources that OLM supports, such as `VerticalPodAutoscalers` and `PrometheusRules`, but the relevant CustomResourceDefinition have not yet been installed. In this case, this error should eventually resolve itself provided the required CustomResourceDefinition gets installed on the cluster and is accepted by the api-server.
```
Comment (exdx) [inline content/en/docs/Tasks/Troubleshooting/subscription.md:45]: nit: CustomResourceDefinition should be CustomResourceDefinitions if we switch to `have`.

I would actually prefer to keep it singular (`has`) and have `installing an operator bundle with a CustomResource...`

File: content/en/docs/Tasks/Troubleshooting/subscription.md [hunk 2]
```diff
@@ -30,15 +30,16 @@ If the `Status` block does not provide enough information, check the [Catalog op
...
 
-In the case where a subscription is created to an operator that relies on missing APIs, for example a v1beta1 CRD, an error condition will be present on the subscription status with the following message: 
+Newer versions of Kubernetes may deprecate and remove certain APIs that OLM supports. For example, the `apiextensions.k8s.io/v1beta1` GroupVersion is no longer available on Kubernetes 1.22+ clusters. Operators that rely on v1beta1 CRDs will fail to install on 1.22+ clusters, since the v1beta API has been removed. See
+[the official Kubernetes deprecation guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/) for the full list of resources that have been deprecated and removed by individual Kubernetes versions. These APIs will continue to be available on older versions of Kubernetes and OLM will continue to support them moving forward.
 
-```
+In the case where a subscription is created to an operator that relies on missing APIs, for example a v1beta1 CRD, an error condition will be present on the subscription status with the following message:
+
+```console
```
Comment (exdx) [inline content/en/docs/Tasks/Troubleshooting/subscription.md:39]: what is the effect of this?

Comment (timflannagan) [reply content/en/docs/Tasks/Troubleshooting/subscription.md:39]: It shouldn't have any user-facing effects - namely for silencing markdown warnings locally and slightly highlights the message more when looking at it through markdown preview.

Comment (timflannagan) [reply content/en/docs/Tasks/Troubleshooting/subscription.md]: Updated to remove the change entirely.

File: content/en/docs/Tasks/Troubleshooting/subscription.md (hunk 1 above)
Comment (timflannagan) [reply content/en/docs/Tasks/Troubleshooting/subscription.md]: Yeah good catch - glossed over that too quickly locally. Reverted back to the original changes as I'm mainly interested in fixing the `offical` -> `official` :laughing:.

**Comment by timflannagan** [comment]
The linting test is due to unrelated changes. Opened https://github.com/operator-framework/olm-docs/pull/175 to fix that link.

**Comment by timflannagan** [comment]
(clearing out my open PRs) - manually merging as this is a safe change that largely only fixes a misspelling that we didn't want to block a PR from merging last time.

**Pull request merged**